http/engine.go

```diff
@@ -277,6 +272,23 @@ func (h Engine) applyBindMiddleware(echoServer EchoServer, path string, excludeP
 		return false
 	}
 
+	// Logging
+	loggerSkipper := func(c echo.Context) bool {
+		// Aside from interface-driven skipper, skip logging for metrics and status
+		if skipper(c) {
+			return true
+		}
+		return matchesPath(c.Request().RequestURI, "/metrics") || matchesPath(c.Request().RequestURI, "/status")
+	}
+	echoServer.Use(requestLoggerMiddleware(func(c echo.Context) bool {
+		// Log when level is set to metadata or request-reply
+		return cfg.Log == LogNothingLevel || loggerSkipper(c)
+	}, log.Logger()))
+	echoServer.Use(bodyLoggerMiddleware(func(c echo.Context) bool {
+		// Log when level is set to request-reply
+		return cfg.Log != LogRequestReplyLevel || skipper(c)
+	}, log.Logger()))
```

Comment:
when `cfg.Log == LogNothingLevel`, no request will ever be logged
when `cfg.Log != LogRequestReplyLevel`, no body will ever be logged

Do we even need to register the middleware in these cases?

Reply:
Probably not.